src/coin/skycoin/models/wallet.go

```diff
@@ -611,13 +900,254 @@ func (wlt *LocalWallet) SetLabel(wltName string) {
 	wlt.Label = wltName
 
 }
-func (wlt *LocalWallet) Transfer(to core.Address, amount uint64) {
+func (wlt *LocalWallet) Transfer(to core.Address, amount uint64, options core.KeyValueStorage) (core.Transaction, error) {
```

Comment:
If transaction creation methods are not substantially different for remote and local wallets then they should be merged into a single function that gets invoked by both methods .